### PR TITLE
Enabling travis build for ppc64le

### PR DESCRIPTION
**Pull request by Ryan-Dmello**
fixes https://github.com/kiali/kiali/issues/2974

**Enable travis build for ppc64le arch**

** Backwards incompatible? **

- Is your change introducing backwards incompatible changes? **No**

**Enabling travis build for ppc64le arch**

** Documentation **

* Does your contribution contain user-visible changes like e.g. new or changed configuration settings?
If so, please also update README.adoc

### CODE REVIEW

**Comment by Ryan-Dmello** [comment]
@jmazzitelli, please find the recent build https://travis-ci.org/github/kiali/kiali/jobs/706404872 it passed.

**Comment by jmazzitelli** [comment]
@Ryan-Dmello I created a github issue to track this work here: https://github.com/kiali/kiali/issues/2974

Note that before we merge this, we'll need the Kiali Operator to also support the new hardware. So you'll need another PR for the kiali-operator project so it also builds with ppc64le.

We did something similiar for the arm64 architecture - see that PR to learn how that was done and you can probably use that as help to get ppc64 builds. That PR is https://github.com/kiali/kiali-operator/pull/6/files

**Comment by jmazzitelli** [comment]
don't merge yet until the travis build errors in the operator are fixed - see https://github.com/kiali/kiali-operator/pull/72

**Comment by jmazzitelli** [comment]
The operator can now build on the new arch (travis confirms). We can merge this now.

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +39,7 @@ env:
...
   - secure: "gYC5NDHZBMmTIPVzxXwZBFJ99UFnrnqN9YTNz3OxrN1qSrWi3g4nNhP2lyreKhaAg5IchYvHwjgktWYE25yCvsMLfaQQXw/2wCwRnEpcMDsVzZxVfExvy8bukhKlgKr8PXU+4mMTo1E+4b+HWdCTMQf0hKt6/7ccF8udY+VxXqQ6Kg9CrRSg3MtsMJR75rOUpJ2KgNG9N5GzCNl3hxwM+F7DpHEp0YstpAWqlugTHhsKGXEUfu/DGhRWfEmQnj0u8ra3qkjsRlanLcWdCfHR4lqPVuRRkeIP3QQkgJe283xVgt6t1sIUdMOSvoSqi+T469WI+nuO9ssPBikViQSc1P2vOay1p3dIx8PKmIvRlDr+fT5d8FvTafWCDMHOrAEJ3drkbjuMo/E3OXNq3QS8aXlFc17raa+gxihSQNVV1weyifng9+IO5JRoi1B9unOwuAl71Xt34uMAjCWJ2tZS/w4c5IPUaSlaXO4ULEgn0YEjxFnrroxBexFkWlgnT8wrjOg3JUOSbNSB1X81IoSEYiDg+Ddt/zn8lS0fQwAMvsuAODdY5EnMbLFDUKOEZxMl6GVAvDzp0LxGfzFLlHTGkPBy92Ox/59qRLXVvUcCn++Dl9TvnI5g8mSTcWjmDDHyTSShzfO/y/+aMc2mQ/OEYZ7u5o8qHOgeqFjrzHN72P0="
+  - SWAGGER_VERSION=0.22.0
```
[the remark this answers is not in the document]
Comment (Ryan-Dmello) [reply .travis.yml]: Made suggested change

File: README.adoc [hunk 2]
```diff
@@ -86,6 +86,7 @@ cd src/github.com/kiali
 git clone git@github.com:kiali/kiali
 git clone git@github.com:kiali/kiali-operator kiali/operator
 export PATH=${PATH}:${GOPATH}/bin
+export SWAGGER_VERSION=0.22.0
```
[the remark this answers is not in the document]
Comment (Ryan-Dmello) [reply README.adoc]: Made suggested change

File: make/Makefile.build.mk [hunk 3]
```diff
@@ -86,7 +86,11 @@ run:
 ## swagger-install: Install swagger from github
 swagger-install:
 	@echo "Installing swagger binary to ${GOPATH}/bin..."
-	@curl https://github.com/go-swagger/go-swagger/releases/download/v0.22.0/swagger_linux_${GOARCH} --create-dirs -Lo ${GOPATH}/bin/swagger && chmod +x ${GOPATH}/bin/swagger
+ifeq ($(GOARCH), ppc64le)
+	curl https://github.com/go-swagger/go-swagger/archive/v${SWAGGER_VERSION}.tar.gz --create-dirs -Lo /tmp/v${SWAGGER_VERSION}.tar.gz && tar -xzf /tmp/v${SWAGGER_VERSION}.tar.gz -C /tmp/ && src_dir='pwd' && cd /tmp/go-swagger-${SWAGGER_VERSION} && go install ./cmd/swagger && cd ${src_dir} && chmod +x ${GOPATH}/bin/swagger
```
[the remark this answers is not in the document]
Comment (Ryan-Dmello) [reply make/Makefile.build.mk]: Made suggested change

**Comment by Ryan-Dmello** [comment]
Thanks @jmazzitelli for the help.

**Pull request merged**